Assets/Scripts/GroundPound.cs

```diff
@@ -0,0 +1,92 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class GroundPound : MonoBehaviour
+{
+
+    public float dropForce =10f;
+    public float stopTime = 0.5f;
+    public float gravityScale = 1f;
+    private PlatformerController player;
+    private Rigidbody2D body;
+    private bool doGroundPound = false;
+    //get animator later
+
+    private void Awake()
+    {
+        player = GetComponent<PlatformerController>();
+        body = GetComponent<Rigidbody2D>();
+    }
+
+    void Update()
+    {
+        //works
+        if (Input.GetKeyDown(KeyCode.S) || Input.GetKeyDown(KeyCode.DownArrow))
```

Comment:
So because we are using the new input system, instead of running the function from an if statement in Update. Provide a public function instead